lib/tilelivecopy.js

```diff
@@ -7,11 +7,12 @@ var combiner = require('stream-combiner');
 var MigrationStream = require('./migration-stream');
 
 function tilelivecopy(srcUri, s3url, options, callback) {
-  var scheme = 'scanline';
+
+  options.type = options.scheme || 'scanline';
+  if(['put', 'pyramid', 'scanline', 'list'].indexOf(options.type) === -1) return callback(new Error('Invalid scheme. Must be "put", "pyramid", "scanline", or "list".'));
```

Comment:
Grabbed it from this list: https://github.com/mapbox/tilelive/blob/master/lib/tilelive.js#L476 but yes you are correct it isn't! Confirmed an error is thrown from tilelive.